lib/browser/chrome-languages.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+// https://developer.chrome.com/webstore/i18n?csw=1#localeTable
+
+module.exports = new Set([
```

Comment:
I think it should be possible to deduplicate this. This list already exists in the chrome source code, we should not be copy/pasting it.

Reply:
re-implemented as discussed 1:1 on Slack
> i’d also accept just enforcing that the path component must be a single component and not escape the directory in any way

Reply:
it's actually not necessary to send the `language` over IPC, we can detect it in the main process